packages/x-data-grid/src/components/GridRow.tsx

```diff
@@ -140,12 +140,12 @@ const GridRow = React.forwardRef<HTMLDivElement, GridRowProps>(function GridRow(
     ...other
   } = props;
   const apiRef = useGridApiContext();
+  const gridConfiguration = useGridConfigurationContext();
```

Comment:
I would rename to use:
```js
const configuration = useGridConfiguration()
```
I find it a bit verbose in its current format.

Reply:
I agree. Updated

Reply:
And `gridConfiguration` to `configuration` as well. The grid prefix is only for exported stuff, it's too verbose to prefix everything with "grid".